src/main/java/org/example/Athlete.java

```diff
@@ -0,0 +1,140 @@
+package org.example;
+
+import java.util.Objects;
+
+import lombok.*;
+
+//Here are the annotations + override in this class
+@Getter
+@Setter
+@ToString
+public class Athlete {
+
+    String number;
+    String name;
+    String countryCode;
+    String skiTimeResult;
+    String firstShootingRange;
+    String secondShootingRange;
+    String thirdShootingRange;
+    String last3Fields;
+
+    public Athlete(String number, String name, String countryCode, String skiTimeResult, String firstShootingRange, String secondShootingRange, String thirdShootingRange, String last3Fields) {
+        this.number = number;
+        this.name = name;
+        this.countryCode = countryCode;
+        this.skiTimeResult = skiTimeResult;
+        this.firstShootingRange = firstShootingRange;
+        this.secondShootingRange = secondShootingRange;
+        this.thirdShootingRange = thirdShootingRange;
+        this.last3Fields = last3Fields;
+    }
+
+    /**
+     * This method checks all the wrong ways the first, second and thirdShootingRange could be.
+     * If the fields are incorrect, the program stops after a short message related to the problem.
+     */
+    public boolean checkShootingFields() {
+        if ((firstShootingRange == null || firstShootingRange.isEmpty()) || (secondShootingRange == null || secondShootingRange.isEmpty()) || (thirdShootingRange == null || thirdShootingRange.isEmpty())) {
+            System.out.println("The shooting range is null or empty! Please add a real value");
+            return false;
+        }
+        if (firstShootingRange.length() != 5 || secondShootingRange.length() != 5 || thirdShootingRange.length() != 5) {
+            System.out.println("The value must have 5 characters");
+            return false;
+        }
+        for (int i = 0; i < firstShootingRange.length(); i++) {
+            char c = firstShootingRange.charAt(i);
+            if (c != 'x' && c != 'o') {
```

Comment:
throw ValidationExeptions here instead of printing to console. Otherwise, very thorough validation, well done